projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss

```diff
@@ -41,7 +41,9 @@
     pointer-events: none;
   }
 
-  &--active .lg-accordion__heading-icon {
-    transform: rotateX(180deg);
+  &--active {
+    .lg-accordion__heading-icon {
```

Comment:
@Zimann this was the only one I didn't quite get, although I think it may be a lack of understanding on my side, are these the same thing?

Reply:
Yep, it's the same thing. It's just making use of the nested selectors from scss instead of having it chained (.classA .classB -> can be written as : .classA { .classB }